app/javascript/components/src/utils/date/index.js

```diff
@@ -1,10 +1,15 @@
-const formatDate = dateString => {
+export const formatDateTime = dateString => {
   if(isValidDate(dateString))
-    return new Date(dateString + ' UTC').toLocaleString()
+    return new Date(dateString).toLocaleString()
    else 
     return null
 };
 
-const isValidDate = d => isNaN(Date.parse(d)) === false
+export const formatDate = dateString => {
+  if(isValidDate(dateString))
+    return new Date(dateString).toLocaleDateString();
```

Comment:
It seems that toLocaleDateString causes a difference in assumed timezones so a time of '2011-09-01T00:00:00' will result in Wed Aug 31 2011 20:00:00 GMT-0400 (EDT) for example.  This function is currently resulting in the day before due to local timezone change.  toLocaleDateString('en-US', {timeZone: 'UTC'}) might fix it.

Reply:
We want the user to see these times in their timezone, not in UTC. So with that we'll want to make sure we are storing everything in UTC

Reply:
Ok, I ran into an issue the other day where it was posting a date the day before, but running locally now everything seems to be ok.  Sorry for the confusion.